### DISCUSSION TITLE
Bitemporal support

### DISCUSSION THREAD

**Discussion by jfbaro** [neondatabase/neon#4386] [Ideas]
Can neon offer bi-temporal capability (the tenant would pay more for each write and storage, but many cases today would justify the extra cost).

https://aiven.io/blog/two-dimensional-time-with-bitemporal-data

**Comment by knizhnik** [comment]
Neon can support bi-temporal model at the same degree as Vanilla Postgres: you can implement it yourself or use existed extensions. Please look at:
 https://hdombrovskaya.files.wordpress.com/2016/09/bi_temporal_pg_open_2016.pdf
There are uch more extensions adding temporal support to  Postgres. Not all of them can be used in Neon (just because of security reasons). But if you need some particular extension, we will support it.
